### PR TITLE
adds black gloves to fire fighting closets and crate

### PR DESCRIPTION
**Pull request by jughu**
Firefighting gear without FIRE RESISTANT GLOVES?? YOU WILL HURT YOUR HANDS WHILE FIGHTING THE FIRE!!!

:cl:  Hyena
add:  cheap fire resistant black gloves to firefighting closets/crate
tweak: Removes black gloves from cloth crafting and replaces it with grey gloves
tweak: Removes fingerless gloves from cloth if you want fingerless just get black gloves and snip the fingertips off with a wirecutter
/:cl:

### CODE REVIEW

**Comment by Cyberboss** [comment]
No, let's not give everyone access to something that can both remove lights and hide fingerprints

**Comment by jughu** [comment]
I mean tachnically they can craft them by using cloth and cut up bedsheets and paint them

**Comment by jughu** [comment]
Gloves are already common as fuck i dont see a problem with this you can litteraly cut up bedsheets and make black gloves

**Comment by jughu** [comment]
@Cyberboss 1 cut bedsheet with wirecutters = 3 cloth 3 cloth = 1 pair of black gloves so a bedsheet equals black gloves why would this be a problem them if theres litteraly bins full of bedsheets

**Comment by Cyberboss** [comment]
Then let them make it. Less hugbox and the heat resistance from gloves is negligible anyway

**Comment by jughu** [comment]
then whats the big problem fire resistant gloves protect your hand from fire ok

**Comment by ShadowDeath6** [comment]
Don't even bullshit the fact that you should get heat resistant gloves in firefighting crates

Closets have the shitty knockoff brand emergency firesuits that don't even protect from fire fully so that might be something to talk about

**Comment by ExcessiveUseOfCobblestone** [comment]
> create fire-resistant gloves using cloth from torn bedsheets

what

**Comment by jughu** [comment]
Yes you can cut up bedsheets for 3 cloth and you can make black gloves with 3 cloth

**Comment by Mervill** [comment]
make fire gloves that can't remove lights and leave behind prints but resist heat, like cheap insulated gloves

**Comment by Cyberboss** [comment]
Yeah, black gloves aren't your average assistant item.

On Dec 16, 2016 7:38 AM, "Mervill" <notifications@github.com> wrote:

make fire gloves that can't remove lights and leave behind prints but
resist heat, like cheap insulated gloves

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/tgstation/tgstation/pull/22227#issuecomment-267585839>,
or mute the thread

**Comment by jughu** [comment]
Oke but i will remove black gloves from bedsheets them and make them grey instead k?

**Comment by Cyberboss** [comment]
@Mervill Removing lights is tied to fire resistance

The one thing I'm worried about is the potential for greyshits to constantly "le advanced darkness" the station. I am one of those greyshits

**Comment by Cyberboss** [comment]
Go ahead with this though. I think I'll make lightbulbs require a screwdriver to remove (Not counting the light re-placer)

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -32,6 +32,21 @@
...
+	icon_state = "black"
+	item_state = "blackgloves"
+	item_color="black"
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	heat_protection = HANDS
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	transfer_prints = TRUE
 	resistance_flags = 0
 	var/can_be_cut = 1
```
Comment (Cyberboss) [inline code/modules/clothing/gloves/color.dm]: Remove lines 42-48, 50, and 51. They are all inherited from black

**Review by Cyberboss — CHANGES_REQUESTED** [review]
DRY it out

**Comment by Mervill** [comment]
> Go ahead with this though. I think I'll make lightbulbs require a screwdriver to remove (Not counting the light re-placer)

Screwdriver notwithstanding the light replacer is a good idea since it give you good incentive to actually use the damn replacer

**Comment by KorPhaeron** [comment]
Uneeded buff

**Comment by jughu** [comment]
Kor wrecking my dreams like a wrecking ball

**Pull request closed without merging**